README.md

```diff
@@ -30,7 +30,8 @@ If you need additional help consider reaching out with one of the following opti
 
 - Ask a question in the [``#cloud-init`` IRC channel on Libera](https://kiwiirc.com/nextclient/irc.libera.chat/cloud-init)
 - Search the cloud-init [mailing list archive](https://lists.launchpad.net/cloud-init/)
-- Better yet, join the [cloud-init mailing list](https://launchpad.net/~cloud-init) and participate
+* Follow announcements or ask a question on [the cloud-init Discourse forum](https://discourse.ubuntu.com/c/server/cloud-init/)
```

Comment:
Good reference to discourse here, we need to start steering folks to a good platform for Q&A and searchable content which is a better than the mailing list. Good to retain the reference for the mailing list as we'll need to reflect important announcements there and have an an engaged community there as well.